### DISCUSSION TITLE
Cannot cache images due to permission issues

### DISCUSSION THREAD

**Discussion by lucala** [sct/overseerr#3122] [Help] [answered]
I enabled the experimental caching feature, my overseerr instance is running in a docker container and looking at the logs I get the following message:

```
Failed to write image to cache OCyoDhGhxG2ySmATFJOEb2PtciYaz++IXd9hQfgarwY= [Error: EACCES: permission denied, mkdir '/app/overseerr/.next/cache/images'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'mkdir',
  path: '/app/overseerr/.next/cache/images'
}
```

Any ideas what I'm missing?

**Comment by ldshangfeng** [comment, marked as answer]
I have the same problem, but I do this solved.
cd /app/overseerr/.next
chmod -R 777 cache
cd cache
mkdir images
chmod -R 777 images
